clients/admin-ui/src/features/datastore-connections/constants.ts

```diff
@@ -79,10 +85,15 @@ export const CONNECTION_TYPE_LOGO_MAP = new Map<
   [SaasType.DOMO, "domo.svg"],
   [SaasType.DOORDASH, "doordash.svg"],
   [SaasType.FIREBASE_AUTH, "firebase.svg"],
+  [SaasType.FRIENDBUY, "friendbuy.svg"],
+  [SaasType.FRIENDBUY_NEXTGEN, "friendbuy.svg"],
   [SaasType.FULLSTORY, "fullstory.svg"],
+  [SaasType.GOOGLE_ANALYTICS, "google_analytics.svg"],
   [SaasType.HUBSPOT, "hubspot.svg"],
   [SaasType.MAILCHIMP, "mailchimp.svg"],
+  [SaasType.MAILCHIMP_TRANSACTIONAL, "mandrill.svg"],
   [SaasType.OUTREACH, "outreach.svg"],
+  [SaasType.RECHARGE, "recharge.svg"],
```

Comment:
Recharge logos
<img width="949" alt="Screen Shot 2023-02-01 at 5 18 21 PM" src="https://user-images.githubusercontent.com/9755598/216190043-7a94024c-bc18-4ebe-83b2-1b7a6d8250dc.png">

Reply:
This is great. Thanks a lot @pattisdr !

Reply:
Welcome!